sinch/domains/verification/models/requests.py

```diff
@@ -4,6 +4,42 @@
 from sinch.domains.verification.models import VerificationIdentity
 
 
+class ReporPhoneCallVerificationDataTransfromationMixin:
```

Comment:
Check for typos:

1. ReporPhoneCallVerificationDataTransfromationMixin
2. ReporFlashCallVerificationDataTransfromationMixin
3. ReporSMSVerificationDataTransfromationMixin

Reply:
indeed; copy/paste typos
fixed